etc/debian/control

```diff
@@ -0,0 +1,36 @@
+Source: scancode-toolkit
+Section: misc
+Priority: optional
+Maintainer: Debian Python Modules Team <python-modules-team@lists.alioth.debian.org>
+Uploaders:Phillipe Ombredanne<pombredanne@nexb.com>, 
+	  Ayush Jain <aj4ayushjain@gmail.com>
+Build-Depends:  debhelper (>= 10), 
+		dh-python, 
+		python-all, 
+		python-setuptools,
+		python-dev,
+		xz-utils,
+		zlib1g,
+		libxml2-dev,
+		libxslt1-dev,
+		libbz2-1.0 | bzip2
```

Comment:
style suggestion: sort dependencies alphabetically, and use a comma even on last line so that diffs are cleaner if you have to add one to the end later

Reply:
Alright will make the changes